tests/python/gaia-ui-tests/gaiatest/apps/bzlite/app.py

```diff
@@ -0,0 +1,26 @@
+from marionette import Marionette
+from gaiatest.apps.base import Base
+from marionette_driver import By, Wait, expected
+
+class BugzillaLite(Base):
+
+    name = 'Bugzilla Lite'
+    _given_username = (By.CSS_SELECTOR, "#login input[type='email']")
+    _given_password = (By.CSS_SELECTOR, "#login input[type='password']")
+    _button_login = (By.CSS_SELECTOR, "#login input[type='submit']")
+    _app_ready_locator = (By.XPATH, "//div[@class='icon']//span[contains(text(),'Bugzilla Lite')]")
```

Comment:
Okay, the Wait won't work anymore then. This locator is present on the homescreen and after `Base.launch()`, you will be in the in the App's frame. As a consequence, the wait will raise a timeout exception.

I think we can change this locator to: `(By.CSS_SELECTOR, '#content header'). This is a bit generic, but that's the most common element between the pages when you're connected and when you're note.